index.html

```diff
@@ -20,377 +20,273 @@
+        <div class="line"></div>
       </div>
-      <div class="project-btn">
-        <h4>See Project</h4>
-      </div>      
-      <section class="projects">
-
-        
-        <h1>Profesional Art Printing Data</h1>
-        <small>A daily selection of privately personalized reads; 
-          no accounts or sign-ups required. 
-          has been the industry's standard
-        </small>
-        <div class="second-tags">
-         <div class="second-tag">
-          <span>Html</span>
-         </div>
-         <div class="second-tag2">
-          <span>bootstrap</span>
-         </div>
-         <div class="second-tag3">
-          <span>Ruby</span>
-         </div>
-        </div>
-
-        <div class="action">
-          <h3>See Project</h3>
-        </div>
-
-
-
 
-        
-
-      </section>
-      <section class="projects">
-
-        
-        <h1>Profesional Art Printing Data</h1>
-        <small>A daily selection of privately personalized reads; 
-          no accounts or sign-ups required. 
-          has been the industry's standard
-        </small>
-        <div class="third-tags">
-          <div class="third-tag">
-           <span>Html</span>
-          </div>
-          <div class="third-tag2">
-           <span>bootstrap</span>
-          </div>
-          <div class="third-tag3">
-           <span>Ruby</span>
-          </div>
-         </div>
-
-        <div class="action">
-          <h3>See Project</h3>
-        </div>
-
-
-
-
-        
-
-      </section>
-      <section class="projects">
-
-        
-        <h1>Profesional Art Printing Data</h1>
-        <small>A daily selection of privately personalized reads; 
-          no accounts or sign-ups required. 
-          has been the industry's standard
-        </small>
-        <div class="fourth-tags">
-          <div class="fourth-tag">
-           <span>Html</span>
-          </div>
-          <div class="fourth-tag2">
-           <span>bootstrap</span>
+      <div class="card-works">
+        <div class="feature">
+          <div class="img-placeholder">
+            <img src="assets/img-placeholder.png" alt=""/>
           </div>
-          <div class="fourth-tag3">
-           <span>Ruby</span>
-          </div>
-         </div>
-
-        <div class="action">
-          <h3>See Project</h3>
-        </div>
-
-
-
-
-        
-
-      </section>
-      <section class="projects">
-
-        
-        <h1>Profesional Art Printing Data</h1>
-        <small>A daily selection of privately personalized reads; 
-          no accounts or sign-ups required. 
-          has been the industry's standard
-        </small>
-        <div class="fifth-tags">
-          <div class="fifth-tag">
-           <span>Html</span>
-          </div>
-          <div class="fifth-tag2">
-           <span>bootstrap</span>
-          </div>
-          <div class="fifth-tag3">
-           <span>Ruby</span>
+          <div class="right-block">
+            <h1>Multi-Post Stories</h1>
```

Comment:
- [ ] [Optional] I would suggest you please use `hx` tag here like `h2, h3,h4` because only we use the `h1` tag per page and it represents the main heading of the page.